src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -147,4 +151,19 @@ public boolean equals(Object obj) {
                 && filteredPersons.equals(other.filteredPersons);
     }
 
+    //=========== Toggle focus mode state ==================================================================
+
+    //TODO: replace with class
+    @Override
+    public void enterFocusMode(Tag classToFocus) {
+        requireNonNull(classToFocus);
+        focusMode.enter(classToFocus);
```

Comment:
Following the above suggestion, this would be `focusedClass = classToFocus` instead.